src/openapi/streaming/streaming.js

```diff
@@ -120,19 +148,18 @@ function onConnectionStateChanged(nextState) {
             for (let i = 0; i < this.subscriptions.length; i++) {
                 this.subscriptions[i].onConnectionUnavailable();
             }
-
             retryConnection.call(this);
             break;
 
         case this.CONNECTION_STATE_RECONNECTING:
-
+            this.retryCount = 0;

```

Comment:
I thought it would only be necessary on conntected?

Reply:
Yeap, it was overkill. Dropped.